wordpress-mobile/WordPress-Android
# Issue/list row padding

### Fix
Update item row layout for ***Sites*** and ***Me*** to show full-width background when pressed.  Consequently, this also updates the width of the card view for ***Me*** to be evenly spaced.  See the screenshots below for illustration.

![row_padding](https://user-images.githubusercontent.com/3827611/40927307-082db6e2-67e4-11e8-9bb7-beb0fba1be60.png)

### Test
1. Go to ***Sites*** tab.
2. Long-press any row item.
3. Notice background is full-width.
4. To go ***Me*** tab.
5. Long-press any row item.
6. Notice background is full-width.
7. Notice card view is evenly spaced.
